src/platform/tigerlake/tigerlake.x.in

```diff
@@ -500,10 +504,12 @@ SECTIONS
     LONG(LOADADDR(.AlternateResetVector.text));
     LONG(ADDR(.AlternateResetVector.text));
     LONG(SIZEOF(.AlternateResetVector.text));
+#if CONFIG_MULTICORE
```

Comment:
Not sure why interrupt level 2 would NOT be needed in single core unless its for IDC only ?

Reply:
@lgirdwood Aren't alternate reset vectors only needed for multicore?

Reply:
ah yes - it's alternate lvl2 IRQ.